help/en/package/jmri/jmrit/logixng/LogixNGBrowser.shtml

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>JMRI: LogixNG Browser</title>
+  <meta name="author" content="Dave Sand">
+  <meta name="keywords" content="JMRI logixng browser">
+
+  <!--#include virtual="/Style.shtml" -->
+
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+  <div id="mBody">
+    <!--#include virtual="../../../../html/tools/logixng/Sidebar.shtml"-->
+    <div id="mainContent">
+
+      <h1>JMRI: LogixNG Browser</h1>
+
+      <p>The browser provides a text report of the contents of a LogixNG table entry or
+      a Module table entry.  The text can be selected and copied to a clipboard or saved
+      as a text file.  This is useful for sharing a configuration or getting support.</p>
+
+      <div style="margin-left: 2em">
+      <a href="images/Browser.png">
+      <img src="images/Browser.png"
+      alt="LogixNG Browser" width="911" height="267" >
+      </a>
+      </div>
+
+      <h2>Browser Options</h2>
+      <p>The following options control the browser content.</p>
+      <dl>
+        <dt>Print line numbers</dt>
+        <dd>LogixNG definitions can be very long.  Line numbers can help identify an area
+        of the listing. It's particular useful when you ask about LogixNG on the JMRI
+        users list. Other users can then point out exactly in the list there the problem
+        lies by giving the line number.</dd>
+
+        <dt>Print error handling</dt>
+        <dd>LogixNG has several methods for handling errors.  The assigned method can be
+        included in the report.  It will appear on the line following 3 colons.  For
```

Comment:
I recommend changing this to:
`... line following three colons.`

Reply:
I will fix this in phase 2.